tests/data/process-invalid-relative-cwd.json

```diff
@@ -1,5 +1,9 @@
 {
 	"process": {
+		"consoleSize": {
```

Comment:
Could you add a new `tests/data/process-no-consolesize.json` and add that file to the `spec_handler_test` table in `tests/spec_handlers/process_test.c` to ensure we can tolerate a missing `consoleSize` entry.